assets/dev/js/editor/elements/views/container.js

```diff
@@ -181,16 +181,33 @@ const ContainerView = BaseElementView.extend( {
 	},
 
 	/**
-	 * Add a `Save as Template` button to the context menu.
+	 * Insert a new container inside an existing container.
+	 *
+	 * @returns {void}
+	 */
+	addNewContainer() {
+		const containerSelector = ( false === this.getContainer().parent.parent ) ? this.getContainer() : this.getContainer().parent;
```

Comment:
`this.getContainer().parent.parent` only returns false for the top level container.
If you have a sub level container with children,  when clicking "Add Container", would we want it to add a sibling or a child to it?

Reply:
Maybe `container.getParentAncestry` may help

Reply:
@hein-obox

Reply:
@iNewLegend I have used the getParentAncestry to check if the container has parents. Is this what you had in mind, Leonid?

@udidol If I have understood Aviv's feedback, he is happy with the current behaviour of only adding sibling elements.